docs/R-IOC/r05_lists.md

```diff
@@ -0,0 +1,9 @@
+## Lists
+
+Lists are variables that store heterogeneous information in one direction. chapter [5.7](https://rstudio-education.github.io/hopr/r-objects.html#lists) of Garett’s book
```

Comment:
"one direction" is unclear.
@Ning-L Better pasting the source that you give (chapter 5.7):

Lists are like atomic vectors because they group data into a one-dimensional set.

However, lists do not group together individual values; lists group together R objects, such as atomic vectors and other lists.

For example, you can make a list that contains a numeric vector of length 31 in its first element,
a character vector of length 1 in its second element, and a new list of length 2 in its third element.

To do this, use the list function:
```
mylist <- list()
```

`list()` creates a list the same way `c` creates a vector. Separate each element in the list with a comma:

```
mylist <- list(100:130, "R", list(TRUE, FALSE))
mylist
## [[1]]
## [1] 100 101 102 103 104 105 106 107 108 109 110 111 112
## [14] 113 114 115 116 117 118 119 120 121 122 123 124 125
## [27] 126 127 128 129 130
## 
## [[2]]
## [1] "R"
##
## [[3]]
## [[3]][[1]]
## [1] TRUE
##
## [[3]][[2]]
## [1] FALSE
```

Reply:
Yep ! I will cover this part, I didn't touch it yet. It was originally the text in r03 that we split into 4 files (r03 to r06).